### PR TITLE
Replace jna with jnr POSIX

### PR DESCRIPTION
**Pull request by darxriggs**
This adds additional changes to pull request #2.

### CODE REVIEW

**Comment by darxriggs** [comment]
There is the same issue with ci.jenkins.io as in #12. Tests passed but the last cleanup step on Windows seems to have an issue.

**Comment by jglick** [comment]
https://github.com/jenkins-infra/pipeline-library/pull/115 may fix this.

File: src/test/java/hudson/security/PAMSecurityRealmTest.java [hunk 1]
```diff
@@ -25,8 +25,8 @@ public void testLoadUsers() {
             // expected
         }
 
-        String name = LIBC.getpwuid(LIBC.geteuid()).pw_name;
-
+        POSIX api = PosixAPI.jnr();
+        String name = api.getpwuid(api.geteuid()).getLoginName();
         System.out.println(Arrays.asList(sc.userDetails.loadUserByUsername(name).getAuthorities()));
```
Comment (jglick) [inline src/test/java/hudson/security/PAMSecurityRealmTest.java]: Does this seem to work when run interactively on Linux? I do not think the test suite really checks anything on CI.

Comment (darxriggs) [reply src/test/java/hudson/security/PAMSecurityRealmTest.java]: This works locally and shows all the groups my user is in.
And yes, there are no adequate tests for this plugin.
At least I verified in the Jenkins UI with my local user that
a) with a valid password I can log in
b) with an invalid password I cannot log in

**Comment by darxriggs** [comment]
@jvz can you also have a look at this one?

**Pull request merged**